Source/Fuse.Controls.ScrollView/Triggers/WhileScrolling.uno

```diff
@@ -0,0 +1,95 @@
+using Uno;
+using Uno.Collections;
+using Fuse;
+using Fuse.Controls;
+using Fuse.Triggers;
+using Fuse.Elements;
+using Fuse.Gestures;
+using Uno.UX;
+
+namespace Fuse.Triggers
+{
+	/**
+		Active while the user is scrolling the parent @ScrollView.
+
+			<ScrollView>
+				<SolidColor ux:Name="scrollBg" Color="#2A2A2A"/>
+				<StackPanel  ItemSpacing="15">
+					<Each Count="30">
+						<Rectangle  Width="200" Height="40" Color="#F0F1E7"/>
+					</Each>
+				</StackPanel>
+				<WhileScrolling>
+					<Change scrollBg.Color="#838383" Duration="0.5"/>
+				</WhileScrolling>	
+			</ScrollView>
+
+		This example will change the @ScrollView background color while the user scrolls.
+		The trigger will revert when the ScrollView stops, even if the user is still touching the screen.
+	*/
+	public class WhileScrolling : WhileTrigger
+	{
+		ScrollViewBase _scrollView;
+		Element _parent;
+
+		protected override void OnRooted()
+		{
+			base.OnRooted();
+
+			if (Parent is Element) {
+				_parent = Parent as Element;
```

Comment:
You're effectively casting twice here, which is a bit costy in terms of performance. I generally prefer this:
```uno
var foo = bar as Foo;
if (foo != null) 
{
	// do something with foo
}
```

However, in this case I think the following is better:
```uno
_parent = Parent as Element;
if (_parent == null)
{
	Fuse.Diagnostics.UserError("Parent must be an Element", this);
	return;
}
```
...because overwriting `_parent` with `null` here seems harmless.